packages/playground/src/utils/mintings.ts

```diff
@@ -0,0 +1,87 @@
+export interface receiptInterface {
+  type: string;
+  node_type?: string;
+  node_id?: number;
+  farm_name?: string;
+  farm_id?: number;
+  measured_uptime?: number;
+  period?: {
+    start: number;
+    end: number;
+  };
+  cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
+  };
+  resource_units?: {
+    cru: number;
+    mru: number;
+    hru: number;
+    sru: number;
+  };
+  stellar_payout_address?: string;
+  tft_connection_price?: number;
+  minted_cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
+  };
+  correct_cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
+  };
+  fixup_cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
+  };
+  correct_tft?: number;
+  minted_tft?: number;
+  fixup_tft?: number;
+  tft?: number;
+  musd?: number;
+  fixupReward?: number;
+}
+
+let hashReceipts: receiptInterface;
+export async function getMintingData(hash: number) {
+  await fetch(`${window.env.MINTING_URL}/api/v1/receipt/${hash}`).then(async res => {
+    const receipt = await res.json();
+    if (receipt.Minting) {
+      const data = {
```

Comment:
Why not have 3 interfaces for the receipt hash instead of re-assigning the values to a new object?

1- An interface for common fields between the minting and fixup.
2- Minting interface, This should extend the common one and have in addition the related fields to the minting.
3- Fixup interface, This should also extend the common one and have in addition the related fields to the fixup.